ansible/roles/vm_set/tasks/main.yml

```diff
@@ -237,15 +246,9 @@
 
 - name: Renumber topology
   include_tasks: renumber_topo.yml
-  when: action == 'renumber_topo'
-
-- name: Connect VMs
-  include_tasks: connect_vms.yml
-  when: action == 'connect_vms'
-
-- name: Disconnect VMs
-  include_tasks: disconnect_vms.yml
-  when: action == 'disconnect_vms'
+  when:
+    - action == 'renumber_topo'
+    - container_type is defined and container_type == "PTF"
```

Comment:
can move this also inside the 'vm_required == True' block and get rid of the 'container_type' var

Reply:
let me do that...